src/main/java/ua/softserveinc/tc/controller/manager/BookingEditController.java

```diff
@@ -110,6 +110,27 @@ public String dailyBookingsByState(@PathVariable String date,
     }
 
     /**
+     * Counting number of children in the selected room for appropriate date.
+     *
+     * @param date current date for counting active children
+     * @param id selected room id
+     * @return number of active children
+     * */
+
+    @GetMapping(value = "dailyBookings/{date}/{id}")
```

Comment:
I think that name is not the best choice for method that must find the amount of children.